hue/hue/base/hue-dc.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
```

Comment:
Should we use `Deployment` instead of DC since (IIUC) Deployment is now preferred by OpenShift?

Reply:
It would definitely simplify things, I'll look into that. The DC was chosen to mirror Internal DH's and the legacy ODH setup. Would you rather see it changed now and here or in a consecutive PR?

Reply:
I personally am fine with having the change in this PR, up to you

Reply:
Changed to `Deployment`